backend/src/models/User.js

```diff
@@ -109,4 +109,8 @@ module.exports = {
     type: 'boolean',
     default: false,
   },
+  locale: {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-17T10:48:50Z" title="Thursday, October 17th 2019, 12:48:50 pm +02:00">Oct 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -109,4 +109,8 @@ module.exports = {
     type: 'boolean',
     default: false,
   },
+  language: {
```


If `Vuex-i18n` calls it `locale` maybe we want to call it `locale` in the backend, too?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-17T16:35:15Z" title="Thursday, October 17th 2019, 6:35:15 pm +02:00">Oct 17, 2019</time>_

---


ja